### DISCUSSION TITLE
Save & Restart in Configuration editor not working

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#15219] [Ask A Question] [answered]
FYI:
Running Frigate 15 Beta 2 in Home Assistant
Save & Restart in Configuration editor not working unless **Watchdog** is enabled in Frigate Add-On Info page.
Frigate will shutdown but not restart.

81Xats

**Comment by NickM-27** [comment, marked as answer]
that is working as expected, restart requires frigate to shutdown, it has to be started by the system
